audio-api/src/main/scala/no/ndla/audioapi/controller/TranscriptionController.scala

```diff
@@ -0,0 +1,171 @@
+  this: Props & TapirController & ReadService & TranscriptionService =>
+  val transcriptionController: TranscriptionController
+  class TranscriptionController() extends TapirController {
+
+    override val serviceName: String         = "transcription"
+    override val prefix: EndpointInput[Unit] = "audio-api" / "v1" / serviceName
+
+    private val videoId   = path[String]("videoId").description("The video id to transcribe")
+    private val audioName = path[String]("audioName").description("The audio name to transcribe")
+    private val audioId   = path[Long]("audioId").description("The audio id to transcribe")
+    private val language  = path[String]("language").description("The language to run the transcription in")
+    private val maxSpeaker =
+      query[Int]("maxSpeaker").description("The maximum number of speakers in the video").default(2)
+    private val format = query[String]("format").description("The format of the audio file").default("mp3")
+
+    def postExtractAudio: ServerEndpoint[Any, Eff] = endpoint.post
+      .summary("Extract audio from video")
+      .description("Extracts audio from a Brightcove video and uploads it to S3.")
+      .in(videoId)
+      .in(language)
+      .in("extract-audio")
+      .errorOut(errorOutputsFor(400, 500))
+      .requirePermission(AUDIO_API_WRITE)
+      .serverLogicPure { _ =>
+        { case (videoId, language) =>
+          transcriptionService.extractAudioFromVideo(videoId, language) match {
+            case Success(_)  => Right(())
+            case Failure(ex) => returnLeftError(ex)
+          }
+        }
+      }
+
+    def getAudioExtraction: ServerEndpoint[Any, Eff] = endpoint.get
+      .summary("Get audio extraction status")
+      .description("Get the status of the audio extraction from a Brightcove video.")
+      .in(videoId)
+      .in(language)
+      .in("extract-audio")
+      .errorOut(errorOutputsFor(400, 500))
+      .requirePermission(AUDIO_API_WRITE)
+      .serverLogicPure { _ =>
+        { case (videoId, language) =>
+          transcriptionService.getAudioExtractionStatus(videoId, language) match {
+            case Success(_)  => Right(())
+            case Failure(ex) => returnLeftError(ex)
+          }
+        }
+      }
+
+    def postTranscription: ServerEndpoint[Any, Eff] = endpoint.post
+      .summary("Transcribe video")
+      .description("Transcribes a video and uploads the transcription to S3.")
+      .in("video")
+      .in(videoId)
+      .in(language)
+      .in(maxSpeaker)
+      .errorOut(errorOutputsFor(400, 500))
+      .requirePermission(AUDIO_API_WRITE)
+      .serverLogicPure { _ =>
+        { case (videoId, language, maxSpeakerOpt) =>
+          transcriptionService.transcribeVideo(videoId, language, maxSpeakerOpt) match {
+            case Success(_) => Right(())
+            case Failure(ex: JobAlreadyFoundException) =>
+              returnLeftError(ex)
+            case Failure(ex) => returnLeftError(ex)
```

Comment:
Denne special-casen gjør ingenting?
```suggestion
            case Failure(ex) => returnLeftError(ex)
```